ssc/cmod_utilityrate5.cpp

```diff
@@ -1394,10 +1394,10 @@ class cm_utilityrate5 : public compute_module
 				size_t cnt;
 				ssc_number_t * ts_sr, *ts_br;
 				ts_sr = as_array("ur_ts_sell_rate", &cnt);
-				ts_br = as_array("ur_ts_buy_rate", &cnt);
 				size_t ts_step_per_hour = cnt / 8760;
 				if (ts_step_per_hour < 1 || ts_step_per_hour > 60 || ts_step_per_hour * 8760 != cnt)
 					throw exec_error("utilityrate5", util::format("invalid number of sell rate records (%d): must be an integer multiple of 8760", (int)cnt));
+				ts_br = as_array("ur_ts_buy_rate", &cnt);
```

Comment:
separated buy rate error?

Reply:
I was actually pushed to make this change by a pysam user who had an acceptable sell rate `ur_ts_buy_rate` but no sell rate. 

The original formulation will check the count of the sell rate but then throw an error about the buy rate, so I moved the checks to be appropriate for the error.